src/Shared/BrowserTesting/src/BrowserManager.cs

```diff
@@ -46,7 +46,7 @@ private async Task InitializeAsync()
 
             async Task InitializeCore()
             {
-                Playwright = await PlaywrightSharp.Playwright.CreateAsync(_loggerFactory, debug: "pw:api");
+                Playwright = await PlaywrightSharp.Playwright.CreateAsync(_loggerFactory/*, debug: "pw:api"*/);
```

Comment:
@javiercn looks like there's some runs where the logging is blowing up trying to write to the log after the test has finished, I'm going to turn off the verbose logging for now to see if that makes things more stable if it does, I'll file an issue where we can debug/fix what's going on (if we want to run with full debug logs on helix anyways):

```
  [Test Assembly Cleanup Failure (BlazorTemplates.Tests.dll)]: PlaywrightSharp.PlaywrightSharpException : Connection closed (System.InvalidOperationException: There is no currently active test.
   at Xunit.Sdk.TestOutputHelper.GuardInitialized() in C:\Dev\xunit\xunit\src\xunit.execution\Sdk\Frameworks\TestOutputHelper.cs:line 51
   at Xunit.Sdk.TestOutputHelper.QueueTestOutput(String output) in C:\Dev\xunit\xunit\src\xunit.execution\Sdk\Frameworks\TestOutputHelper.cs:line 66
   at Microsoft.Extensions.Logging.LoggerExtensions.Log(ILogger logger, LogLevel logLevel, EventId eventId, Exception exception, String message, Object[] args)
   at Microsoft.AspNetCore.BrowserTesting.PageInformation.RecordConsoleMessage(Object sender, ConsoleEventArgs e) in /_/src/Shared/BrowserTesting/src/PageInformation.cs:line 107
   at PlaywrightSharp.Transport.Channels.PageChannel.OnMessage(String method, Nullable`1 serverParams)
   at PlaywrightSharp.Transport.Connection.Transport_MessageReceived(Object sender, MessageReceivedEventArgs e) in /Users/neo/Documents/Coding/Personal/playwright-sharp/src/PlaywrightSharp/Transport/Connection.cs:line 332)
  Stack Trace:
     at PlaywrightSharp.Transport.Connection.SendMessageToServerAsync[T](String guid, String method, Object args, Boolean ignoreNullValues, JsonSerializerOptions serializerOptions, Boolean treatErrorPropertyAsError) in /Users/neo/Documents/Coding/Personal/playwright-sharp/src/PlaywrightSharp/Transport/Connection.cs:line 163
```

Reply:
Runs are much happier with all the logging turned down, I think we should leave this setting off for now

Reply:
We have the philosophy to capture everything by default, otherwise, when things fail we don't have a way to diagnose it. Maybe we need to figure out an alternative login instead of shoving it all to the diagnostics message sink in one go.

Reply:
That's fine, but I'm just saying this test won't last a day if we try to merge this with that debug flag on, it will be quarantined immediately.  If these tests end up still being too flaky and get quarantined, then we can turn on the debugging as part of the stabilization anyway.  I'll add this debugging logging issue to the clean up tracking issue for this PR.